proxy/proxy.go

```diff
@@ -164,8 +150,11 @@ func NewProxy(cfg *Config, backend *storage.RedisClient) *ProxyServer {
 func (s *ProxyServer) Start() {
 	log.Printf("Starting proxy on %v", s.config.Proxy.Listen)
 	r := mux.NewRouter()
-	r.Handle("/{login:0x[0-9a-fA-F]{40}}/{id:[0-9a-zA-Z-_]{1,8}}", s)
-	r.Handle("/{login:0x[0-9a-fA-F]{40}}", s)
+	/*
+		// Removed HTTP compatibility for now. Miners generally use TCP.
```

Comment:
comments? do we need it?

Reply:
I can remove it and create an issue to track.

Reply:
https://github.com/dominant-strategies/go-quai-stratum/issues/11

Reply:
Removed in next commit